robot-mode.el

```diff
@@ -194,6 +201,10 @@ Used as `indent-line-function' of the mode."
 	    (string-width (buffer-substring-no-properties
 			   (line-beginning-position) (point)))))
 
+	 ;; Current point in line as width
+	 (point-in-line (string-width (buffer-substring-no-properties
+			   (line-beginning-position) (point))))
+
```

Comment:
`(current-column)` gives you the same result

Reply:
Thanks! I didn't know about this. Unfortunately Emacs documentation says that it was introduced around version 29.1. I would like to retain support for versions older than that.

Reply:
I can confirm that it is available on emacs 27.1. It's a built-in function compiled in the C source. The emacs documentation can't really get the exact introduction of these kind of function it seems.
 
And in my opinion, emacs 27.1 is currently the version of the "oldstable" package of debian. So, we can consider it as outdated.